### ISSUE TITLE
`playwright/missing-playwright-await` does not detect `Promise.race`

### ISSUE THREAD

**Issue by Iaotle** [playwright-community/eslint-plugin-playwright#444]
I have to ignore this rule when awaiting inside of a promise race:
```js
await Promise.race([
        // eslint-disable-next-line playwright/missing-playwright-await
        page.waitForRequest((request) => {
          return request.url().includes(`/admin/content/pages/${pageId}/lock`) && request.method() === 'POST';
        }),
        // ...
]);
```
